test/integration/targets/inventory_vmware_vm_inventory/runme.sh

```diff
@@ -41,11 +44,14 @@ echo "Kill all previous instances"
 curl "http://${VCENTER_HOST}:5000/killall" > /dev/null 2>&1
 
 echo "Start new VCSIM server"
-curl "http://${VCENTER_HOST}:5000/spawn?datacenter=1&cluster=1&folder=0" > /dev/null 2>&1
+curl "http://${VCENTER_HOST}:5000/spawn?dc=2&cluster=1&folder=0" > /dev/null 2>&1
 
 echo "Debugging new instances"
 curl "http://${VCENTER_HOST}:5000/govc_find"
 
+# Creates folder structure to test inventory folder support
+ansible-playbook -i 'localhost,' test_vmware_prep_folders.yml
+
```

Comment:
I could not figure out how to use the folder param of vcsim to get a folder structure that I could test for. It seems kindof broken to me. So I prepare a folder structure using this preparation playbook. I took care of making Ansible not to thrown a warning about wrong inventory or implicit localhost, which included the change in ansible.cfg to remove the hardcoded inventory path (which was actually not needed already) and add host_list plugin.